hq/test/aws/ec2/EC2Test.scala

```diff
@@ -115,6 +116,26 @@ class EC2Test extends FreeSpec with Matchers with Checkers {
     }
   }
 
+  "security group" - {
+    "sort by usage" in {
+      val sgsOpenPorts = for {
+        status <- Gen.oneOf("Ok", "Warning", "Error")
+        name <- Gen.alphaStr
+        id <- Gen.alphaStr
+        vpcId <- Gen.alphaStr
+        port <- Gen.oneOf(0 to 65000)
+        instanceId <- Gen.alphaStr
+        usage <- Gen.someOf(Seq(Instance(instanceId), UnknownUsage("unknown", "nic-1"), ELB(instanceId)) )
+      } yield SGOpenPortsDetail(status, "eu-west-1", name, id, vpcId, "tcp", port.toString, "Yellow", false ) -> usage.toSet[SGInUse]
+
+      forAll(Gen.listOf(sgsOpenPorts)) { detail =>
+        val sortedResult = EC2.sortSecurityGroupsByInUse(detail)
+        sortedResult should be (sortedResult.sortWith{ case  ((_, s1), (_, s2)) => s1.size > s2.size })
```

Comment:
Does this test offer any value given that it is literally a copy paste of the implementation?

Reply:
yes, it checks sort is stable. 
e.g. if two items has same priority their place shouldn't be changed by any sort.

Reply:
But I'm talking about stable between accounts which is a different thing, not stable based on whatever order AWS provides them. Are MongoDB SGs always above Postgres, for example. It'll make it easier for InfoSec to review accoutns if they are always displayed the same way. This isn't a hard requirement now but I think it'd be useful. It'll be easier to implement now then change it later! If you don't agree this is important then no worries (or we can chat IRL, whatever you like).

Reply:
i think that's doable, provided we put MongoDB before Postgres in the priority list. 
For that to work, I guess I can use an index based priority map so that each entry would have a unique position.